.rubocop.yml

```diff
@@ -13,3 +13,12 @@ Style/DocumentationMethod:
 
 Style/ClassAndModuleChildren:
   Enabled: false
+
```

Comment:
I believe that trailing commas are both common and should be preferred. Do you see otherwise?